src/transport/FabricTable.h

```diff
@@ -123,7 +123,7 @@ class DLL_EXPORT FabricInfo
     }
 
     CHIP_ERROR GetCredentials(Credentials::OperationalCredentialSet & credentials, Credentials::ChipCertificateSet & certSet,
-                              Credentials::CertificateKeyId & rootKeyId);
+                              Credentials::CertificateKeyId & rootKeyId, uint8_t & credentialsIndex);
```

Comment:
Please add documentation for these: what does the credentialsIndex returned by this method mean?

In a previous comment I was wondering if this could be removed because if this just means "index of last valid credential in the credential set" then it seems redundant information.

Reply:
Updated.